openapi.yaml

```diff
@@ -2386,6 +2386,41 @@ paths:
           $ref: '#/components/responses/client_error_auth'
         5XX:
           $ref: '#/components/responses/server_error'
+  /processing_options:
+    get:
+      summary: Additional processing options
+      operationId: list-processing-options
+      description: |-
+        Lists additional custom processing options that a back-end offers for the
+        different processing modes (synchronous processing, batch jobs, secondary web services).
+
+        The options specified here can be added to the corresponding `POST` requests
+        at the top-level of the object sent in the payload.
```

Comment:
Maybe also make it clear that  all options, being "additional", should be optional (or have default values), 
in the sense  that a batch job will still be accepted when none of these options are specified at job create time.
Or differently put: a backend should not define a _required_ additional parameter.

Reply:
A back-end could do that actually. Normally everything should be optional, but if one chooses to require something here, it is free to do so. But I'll add some text to give some more context...

Reply:
> but if one chooses to require something here, it is free to do so

Isn't that a breaking change in some way then?

Reply:
For their implementation, it might be breaking, yes. Every implementation is free to break whatever they like in their implementation ;-) But that's independent of the API spec and for the API spec it's not a breaking change, no.

Reply:
Added:

> All options SHOULD explicitly be made optional with reasonable defaults
> as otherwise the interoperability between the implementations decreases.